lib/std/crypto/kyber.zig

```diff
@@ -769,22 +762,20 @@ fn computeZetas() [128]i16 {
 const Poly = struct {
     cs: [N]i16,
 
-    const packed_length = N / 2 * 3;
+    const compressed_length = N / 2 * 3;
     const zero: Poly = .{ .cs = .{0} ** N };
 
     fn add(a: Poly, b: Poly) Poly {
         var ret: Poly = undefined;
-        var i: usize = 0;
-        while (i < N) : (i += 1) {
+        inline for (0..N) |i| {
```

Comment:
I think it's better not to use `inline` here. The compiler is clever enough to unroll this loop if it sees fit; we don't have to force it, and code size is important.

Reply:
de-inlined :)